dev/NavigationView/NavigationView.cpp

```diff
@@ -379,10 +427,354 @@ void NavigationView::OnApplyTemplate()
     UpdateBackAndCloseButtonsVisibility();
     UpdateSingleSelectionFollowsFocusTemplateSetting();
     UpdateNavigationViewUseSystemVisual();
-    PropagateNavigationViewAsParent();
     UpdatePaneVisibility();
     UpdateVisualState();
     UpdatePaneTitleMargins();
+    SyncItemTemplates();
+}
+
+void NavigationView::UpdateRepeaterItemsSource(bool forceSelectionModelUpdate)
+{
+    auto dataSource = MenuItemsSource();
```

Comment:
try to avoid the name dataSource. It was a name we decided not to use - Just use itemsSource to avoid confusion